src/genophenocorr/model/_cohort.py

```diff
@@ -299,6 +276,7 @@ def get_excluded_ids(self):
     def get_excluded_count(self):
         return len(self.all_excluded_patients)
 
+    """
```

Comment:
The block is commented out in a very creative way!

It would be probably better to use `#` to comment out the function, if, for some reason, you prefer to keep the code around.